platform/pulp/platform/models/base.py

```diff
@@ -51,7 +51,7 @@ class MasterModel(Model):
         subclass.
 
     """
-    type = models.CharField(max_length=63)
+    real_model = models.CharField(max_length=63)
```

Comment:
I like the reasoning, but I think the name should reinforce the master/detail relationship. Instead of `real_model`, how about `detail_model`, or `detail_type`?